src/main/webapp/app/exercises/shared/plagiarism/plagiarism-sidebar/plagiarism-sidebar.component.html

```diff
@@ -15,8 +15,8 @@
                 <span
                     class="plagiarism-status-indicator"
                     [ngClass]="{
-                        confirmed: comparison.status === 'confirmed',
-                        denied: comparison.status === 'denied'
+                        confirmed: comparison.status === 'CONFIRMED',
+                        denied: comparison.status === 'DENIED'
```

Comment:
you should use the defined enum which is defined in PlagiarismStatus.ts

Reply:
I tried to do that, but apparently, I can't use the TypeScript enum in an Angular template. Do you know a way around that?

Reply:
you have to make it available in the Typescript component, using something like
`readonly CONFIRMED = CONFIRMED`
(it's really weird, but otherwise the template does not know that this exists), there are some other places with enums which use this mechanism as well